### DISCUSSION TITLE
IDD A2 Part 1 Feedback

### DISCUSSION THREAD

**Discussion by hgimonet** [hgimonet/sp2021_IDD_Interactive-Lab-Hub#13] [General]
Please leave your feedback for lab 2 here!

**Comment by nicole-zy** [comment]
Hi Hortense! I like your idea of the hourglass. Perhaps, you can change the color of sand/water inside to notify the user about different hours.

**Comment by ruimaki** [comment]
I really like how you made the device interactive so depending on what button you pressed you could reset or add times to the clock. I wasn't exactly sure what the A button represented but even if the hourglass idea visual becomes too complicated, perhaps you could do a box or circle that could fill up. (since that might be easier to code)
